app/helpers/application_helper.rb

```diff
@@ -1,7 +1,12 @@
 module ApplicationHelper
-  def page_title(title)
+  def page_title(title, error_present = false)
```

Comment:
I always find boolean arguments a little confusing, could we use a keyword/named argument to make it a bit more explicit what the boolean is for?

Reply:
Ah, nice. Tweaked that to use a named argument and it makes a lot more sense now 😃